### ISSUE TITLE
An irrelevant exception causes the first test on ESR38 fail

### ISSUE THREAD

**Issue by xabolcs** [mozilla/memchaser#240]
The root cause is that `loop.enabled` is `false` in ESR38.
```
     [exec] console.error: memchaser: 
     [exec]   fail:
     [exec]   There was an uncaught Promise rejection: Error: loop is not enabled @ resource:///modules/loop/MozLoopService.jsm:1110
     [exec] this.MozLoopService.initialize<@resource:///modules/loop/MozLoopService.jsm:1110:29
     [exec] TaskImpl_run@resource://gre/modules/Task.jsm:314:40
     [exec] TaskImpl@resource://gre/modules/Task.jsm:275:3
     [exec] createAsyncFunction/asyncFunction@resource://gre/modules/Task.jsm:249:14
     [exec] LoopUI.init@chrome://browser/content/browser.js:4488:7
     [exec] gBrowserInit._delayedStartup@chrome://browser/content/browser.js:12093:5
     [exec] 
     [exec] console.trace: memchaser: 
     [exec] _ecated/unit-test.js 108 fail
     [exec] _ecated/unit-test.js 71 TestRunner.prototype._uncaughtErrorObserver
     [exec] _/Promise-backend.js 200 PendingErrors.report
     [exec] _/Promise-backend.js 212 PendingErrors.flush
     [exec] _/Promise-backend.js 614 Promise.Debugging.flushUncaughtErrors
     [exec] _ecated/unit-test.js 322 done
     [exec] _ecated/unit-test.js 564 start
     [exec] _ecated/unit-test.js 522 startMany/runNextTest/<
     [exec] _/Promise-backend.js 867 Handler.prototype.process
     [exec] _/Promise-backend.js 746 this.PromiseWalker.walkerLoop
     [exec] _/Promise-backend.js 688 this.PromiseWalker.scheduleWalkerLoop/<

```

**Comment by xabolcs** [comment]
`--prefs` is my friend! :)

```
    --prefs <path>           Custom set user preferences (path to a json file)
```

PR is coming!

**Comment by xabolcs** [comment]
This needs PR #239 to be merged first!

**Comment by xabolcs** [comment]
Fixed by the commit d7be493d56e32cc9ed0f0733aca92cdb18d330da above.
